src/index.js

```diff
@@ -11,6 +11,19 @@ const client = new ApolloClient({
   cache: new InMemoryCache()
 });
 
+const requireFile = require.context(
+  '@snapshot-labs/snapshot-spaces/skins/',
+  true,
+  /[\w-]+\.scss$/
+);
+
+requireFile.keys().map(file => requireFile(file));
+
+requireFile
+  .keys()
+  .map(file => file.replace('./', '').replace('.scss', ''));
```

Comment:
this will load all the snapshot skins into the iframe so they can be reference and override the css variables `var(--bg-color)` etc - we took it straight from the snapshot repo and it is how they load all the skins into their own project.